python/ray/tune/utils/callback.py

```diff
@@ -141,3 +148,31 @@ def _create_default_callbacks(
                 )
 
     return callbacks
+
+
+def _initialize_env_callbacks() -> List[Callback]:
+    """Initialize callbacks from environment variable.
+
+    Returns:
+        List of callbacks initialized from environment variable.
+    """
+    callbacks = []
+    callbacks_str = os.environ.get(RAY_TUNE_CALLBACKS_ENV_VAR, "")
+    if not callbacks_str:
+        return callbacks
+
+    for callback_path in callbacks_str.split(","):
+        callback_path = callback_path.strip()
+        if not callback_path:
+            continue
+
+        try:
+            module_path, class_name = callback_path.rsplit(".", 1)
+            module = importlib.import_module(module_path)
+            callback_cls = getattr(module, class_name)
+            callback = callback_cls()
```

Comment:
Same discussion as for Train I guess? The class should be validated at this point.

Reply:
Added subclass check.